File: backend/services/identity-service/src/main/java/com/lmspilot/identity/IdentityServiceApplication.java
package com.lmspilot.identity;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class IdentityServiceApplication {
    public static void main(String[] args) { SpringApplication.run(IdentityServiceApplication.class, args); }
}

File: backend/services/identity-service/src/main/kotlin/com/lmspilot/identity/IdentityServiceApplication.kt
package com.lmspilot.identity

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class IdentityServiceApplication

fun main(args: Array<String>) {
    runApplication<IdentityServiceApplication>(*args)
}
